core/domain/user_jobs_one_off.py

```diff
@@ -491,6 +468,44 @@ def reduce(key, values):
         yield (key, len(values))
 
 
+class RemoveActivityIDsOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that deletes the activity_ids from the UserSubscriptionsModel. """
```

Comment:
Please add a line explaining that this is a job that is needed just for the February release and can be removed thereafter.